.github/workflows/test-lit.yml

```diff
@@ -0,0 +1,73 @@
+      run: yarn install
+
+    - name: Create .env file for lit-action-using-fetch
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/lit-action-using-fetch/nodejs/.env
+
+    - name: Create .env file for paying-for-lit
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/paying-for-lit/nodejs/.env
+
+    - name: Create .env file for payment-delegation-db-relayer
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/payment-delegation-db-relayer/nodejs/.env
+        echo "LIT_NETWORK=${{ secrets.LIT_NETWORK }}" >> $GITHUB_WORKSPACE/payment-delegation-db-relayer/nodejs/.env
+        echo "LIT_RELAYER_API_KEY=${{ secrets.LIT_RELAYER_API_KEY }}" >> $GITHUB_WORKSPACE/payment-delegation-db-relayer/nodejs/.env
+        echo "LIT_PAYER_SECRET_KEY=${{ secrets.LIT_PAYER_SECRET_KEY }}" >> $GITHUB_WORKSPACE/payment-delegation-db-relayer/nodejs/.env
+
+    - name: Create .env file for pkp-migration
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/pkp-migration/nodejs/.env
+        echo "LIT_FROM_NETWORK=${{ secrets.LIT_FROM_NETWORK }}" >> $GITHUB_WORKSPACE/pkp-migration/nodejs/.env
+        echo "LIT_TO_NETWORK=${{ secrets.LIT_TO_NETWORK }}" >> $GITHUB_WORKSPACE/pkp-migration/nodejs/.env
+        echo "ADD_PKP_ETH_ADDRESS_AS_PERMITTED_ADDRESS=${{ secrets.ADD_PKP_ETH_ADDRESS_AS_PERMITTED_ADDRESS }}" >> $GITHUB_WORKSPACE/pkp-migration/nodejs/.env
+        echo "SEND_PKP_TO_ITSELF=${{ secrets.SEND_PKP_TO_ITSELF }}" >> $GITHUB_WORKSPACE/pkp-migration/nodejs/.env
+
+    - name: Create .env file for session-signatures/getLitActionSessionSigs
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/session-signatures/getLitActionSessionSigs/.env
+
+    - name: Create .env file for session-signatures/getPkpSessionSigs
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/session-signatures/getPkpSessionSigs/.env
+
+    - name: Create .env file for session-signatures/getSessionSigs
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/session-signatures/getSessionSigs/.env
+
+    - name: Create .env file for sign-and-combine-ecdsa
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/sign-and-combine-ecdsa/nodejs/.env
+        echo "CHAIN_TO_SEND_TX_ON=${{ secrets.CHAIN_TO_SEND_TX_ON }}" >> $GITHUB_WORKSPACE/sign-and-combine-ecdsa/nodejs/.env
+
+    - name: Create .env file for wrapped-keys
+      run: |
+        echo "ETHEREUM_PRIVATE_KEY=${{ secrets.ETHEREUM_PRIVATE_KEY }}" > $GITHUB_WORKSPACE/wrapped-keys/nodejs/.env
+        echo "SOLANA_PRIVATE_KEY=${{ secrets.SOLANA_PRIVATE_KEY }}" >> $GITHUB_WORKSPACE/wrapped-keys/nodejs/.env
```

Comment:
Nice! do these private keys hold assets? We might be able to generate them with a script instead of including static keys in GH secrets that we emit to files just in time.

Reply:
These are needed for the tests of all subdirectories, they do need some faucet funds in them to send transactions.

Maybe @spacesailor24 can give more info about this if it is possible.

Reply:
One thing we have done that works pretty well is define a single `private key` with is funded with a large amount of a given token. Then that wallet transfers funds to wallets created just in time for testing. which also helps with `nonce` ordering concerns since each test suite can then have its own EOA.

Reply:
Thanks @joshLong145 this is a really great callout! As is we're almost guaranteed to run into nonce issues if we try to do anything in parallel

I think what you're suggesting is a great approach: a single funding account and randomly generated EOAs for each test when the tests are initialized. For local development, we still need to support specifying a single key that's used for the specific test

Reply:
I think for local testing a similar approach can be taken where a key is just in time generated at start of the test which is funded. However, it could be beneficial to put this behavior behind a flag such that a static key can be used. Since nonce issues will happen locally if tests are run in parallel.

Reply:
@anshss @spacesailor24 
To achieve the single key which funds other keys at start of the test cycle, I think we can use `tinny` which is being split into its own package in a PR I have up [here](https://github.com/LIT-Protocol/js-sdk/pull/549) on the js-sdk repo. 

It will require refactoring to the test suite in this repository to use `tinny` but will give you the same testing abstractions we use in the sdk. which already has this functionality of generating and funding keys, as it abstracts the concept of the asym key pair to a `person`.

For now we can leave the implementation as is in this PR and work towards a better strategy, either with `tinny` or a separate implementation.